assignments/003-illumina-sequencing-questions/illumina-seq-homework.md

```diff
@@ -0,0 +1,58 @@
+your answers in the "dual-choice in parentheses" questions.
+
+When you are done.  Save the file then commit it
+to a branch called `illumina-seq` and push it back to your
+fork of the course repo on GitHub then send me a pull request.
+
+
+
+* Q1: True or False. This animation is showing that each of the two "fragments" is a double-stranded segment of DNA. (_from 30.0 to 33.0 seconds in the video_).
+
+True
+
+* Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
+
+The top is complimentary to the bottom strand. 
+
+* Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
+
+The sequencing binding site is complimentary strand and can bind with the oligonucleotide on the lawn on its 3'-hydroxyl site such that it would be able to synthesize a new copy via a bridge amplification.
+
+* Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
+
+5' end. 
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+
+The remaining strand will be anchored by the 3' end
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+
+The purple part of the remaining strand is the same as the purple lawn oligos. 
+
+* Q7: Choose one from each set of parentheses: the blue part of the remaining strand is at the (3'|5') end and is (the same as|complementary to) the blue oligos on the lawn.  (_from 93.0 to 96.0 seconds in the video_).
+
+5' and this is complimentary to the blue oligos on the lawn.
+
+* Q8: The polymerase grows the new daughter strand from 5' to 3'.  Does that accord with your previous answers?  Make sure your answers are consistent with this. (_from 98.0 to 101.0 seconds in the video_).
+
+The new strands are made in the reverse direction from the 3' end to the 5' end by the polymerase. So, yes this alligns with my previous answers.  
+
+* Q9: Choose one: Every oligo, purple or blue, on the lawn is attached by its (5'|3') end. (_from 105.0 to 109.0 seconds in the video_).
+
+5' end.
+
+* Q10: Choose one: Every blue or purple sequence dangling at the unattached end of a strand is at the (5'|3') end of the strand. (_from 109.0 to 113.0 seconds in the video_).
+
+3'end.
+
+* Q11: Is read 2 sequenced off the same strand as read 1? (_from 240.0 to 250.0 seconds in the video_).
+
+Yes, but it differs in that they are reverse of each other. 
```

Comment:
Right---it is the reverse complement, which actually means it is sequenced off the different strand.